docs-ref-services/latest/hdinsight.md

```diff
@@ -353,7 +353,7 @@ The HDInsight Management SDK can also be used to manage monitoring on your clust
 ### Enable OMS Monitoring
 
 > [!NOTE]
-> To enable OMS Monitoring, you must have an existing Log Analytics workspace. If you have not already created one, you can learn how to do that here: [Create a Log Analytics workspace in the Azure portal](https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-quick-create-workspace).
+> To enable OMS Monitoring, you must have an existing Log Analytics workspace. If you have not already created one, you can learn how to do that here: [Create a Log Analytics workspace in the Azure portal](/azure/log-analytics/log-analytics-quick-create-workspace).
```

Comment:
I think that this changed link does need to include the /en-us/ at the front: "/en-us/azure/log-analytics/log-analytics-quick-create-workspace"